core/index.js

```diff
@@ -0,0 +1,27 @@
+const fs = require("fs");
+const path = require("path");
+
+function getArtifact(contract) {
+  const filePath = path.join(__dirname, "build", "contracts", `${contract}.json`);
+  if (!fs.existsSync(filePath)) {
+    return null;
+  }
+
+  return JSON.parse(fs.readFileSync(filePath));
+}
+
+function getAbi(contract) {
+  const artifact = getArtifact(contract);
```

Comment:
Prob outside scope of this PR, but how do you propose solving problem where devs need to integrate with older ABI's? How do you think we should handle versioning of the ABI's

Reply:
I think this should be managed through _versions_ of the `core` package. For instance, if someone wants to use/integrate an older version of the abi, they should use `@umaprotocol/core@1.0.0` rather than `@umaprotocol/core@latest`, etc. Do you think that seems reasonable?

Reply:
Yeah that's a great point once we've fully package-ized versioning should come mostly free